spec/features/deck_management_spec.rb

```diff
@@ -35,7 +35,7 @@
       create(:deck, user: user)
             
       visit decks_path
-      click_link 'Set current'
+      click_link 'Сделать текущей'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.